test/integrations/destinations/af/processor/data.ts

```diff
@@ -1716,6 +1716,98 @@ export const existingTestCases = [
       },
     },
   },
+  {
+    name: 'af',
+    description: 'Place selected properties at root track call with af data',
+    id: 'selectedPropsAtRoot',
+    feature: 'processor',
+    module: 'destination',
+    version: 'v0',
+    input: {
+      request: {
+        body: [
+          {
+            message: {
+              type: 'track',
+              event: 'Order Completed',
+              sentAt: '2020-08-14T05:30:30.118Z',
+              context: commonContextWithExternalId,
+              messageId: '7208bbb6-2c4e-45bb-bf5b-ad426f3593e9',
+              timestamp: '2020-08-14T05:30:30.118Z',
+              properties: {
+                ...commonPropertiesWithProduct,
+                prop1: 'value1',
+                prop2: 'value2',
+                Prop3: 'value3',
```

Comment:
```suggestion
                prop3: 'value3',
```

Reply:
This is intensional to check case sensitivity.